### PR TITLE
Adding information on timezone based alerting

### PR DESCRIPTION
**Pull request by ShrishaKumar**

### CODE REVIEW

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[8b5f5e7](https://github.com/Azure/portaldocs/pull/191/commits/8b5f5e7baff918893521f9916d2b8c6c35b6c71b)_: 

:clock10: **Preparing**: average preparing time is _45 sec(s)_

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[8b5f5e7](https://github.com/Azure/portaldocs/pull/191/commits/8b5f5e7baff918893521f9916d2b8c6c35b6c71b)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C28%5C3d86255f-f488-188e-02a3-9e92aa67830d%5CPullRequest%5C201902282356282947-191%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=CnJG6yw7%2B6f9f5OUV%2BQaXjcGVH%2F1iocxS22Z0jS8KcA%3D&st=2019-02-28T23%3A51%3A39Z&se=2019-03-31T23%3A56%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: portal-sdk/templates/top-telemetry-alerting.md [hunk 1]
```diff
@@ -700,11 +702,37 @@ Alerts are supported in national clouds. Specify the national cloud portal domai
...
+## Timezone based alerting
+
+As Ibiza has extension developers spread across the world, we have a mechanism to trigger alerts in the business hours of the extension team. Currently, the alerts supported for timezone based alerting are - 
+1. SDK Age alerts.
+
+For all other alerts, the extension owner cannot pick a timezone and will be alerted as soon as the alert trigger conditions are met.
+
+To configure timezone based alerting, you need to specify a `businessHourStartTimeUtc` property in the alerting config. The value takes an integer value from `0` to `23` as a string. The value represents the UTC hour at which business hours start in the extension team's region.
+
+The Ibiza team guarentees that you will receive an alert within 6 hours of the hour configured as `businessHourStartTimeUtc`.
```
Comment (HuaZhao) [inline portal-sdk/templates/top-telemetry-alerting.md]: Append "when an alert is raised"

File: portal-sdk/templates/top-telemetry-alerting.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ There are number of framework provided alerts:
 1. Client Error
 1. Create regression
 
+Note: You can set the time of the day at which you want to trigger an SDK age alert. Learn more about timezone based alerting [here](#timezone-based-alerting).
```
Comment (HuaZhao) [inline portal-sdk/templates/top-telemetry-alerting.md]: Can we move this right below the Extension SDK age? Something looks like:

Extension SDK age
  Sev3 IcM incdient for an extension when its SDK is older than 60 days
  Sev2 for over 90 days
   Note: You can set the time of the day at which you want to trigger an SDK age alert. Learn more about timezone based alerting [here](#timezone-based-alerting).
Extension alive

File: portal-sdk/templates/top-telemetry-alerting.md [hunk 3]
```diff
@@ -700,11 +702,37 @@ Alerts are supported in national clouds. Specify the national cloud portal domai
 }
 ```
 
-[alerting-onboarding]: https://aka.ms/portalfx/alerting-onboarding
```
Comment (HuaZhao) [inline portal-sdk/templates/top-telemetry-alerting.md]: Why removing those hyperlinks?

File: portal-sdk/templates/top-telemetry-alerting.md [hunk 4]
```diff
@@ -700,11 +702,37 @@ Alerts are supported in national clouds. Specify the national cloud portal domai
...
+        ...
+    ]
+    ...
+}
+```
+
+If no value is specified for `businessHourStartTimeUtc`, alerts are triggered in PST business hours by default.
+
 
-[datacenter-code-name]: https://aka.ms/portalfx/alerting/datacenter-code-name
```
Comment (HuaZhao) [inline portal-sdk/templates/top-telemetry-alerting.md]: Why removing those links? They're being used in the doc, no?

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[4b49890](https://github.com/Azure/portaldocs/pull/191/commits/4b49890778d7c224434c3aae23bd85ce61a5c42a)_: 

:clock10: **Preparing**: average preparing time is _45 sec(s)_

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[4b49890](https://github.com/Azure/portaldocs/pull/191/commits/4b49890778d7c224434c3aae23bd85ce61a5c42a)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C3d86255f-f488-188e-02a3-9e92aa67830d%5CPullRequest%5C201903010039079894-191%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=PppBl7J1nxaf5Z18Qz5hn4ltcPXLO5MmGngJYbpsdy4%3D&st=2019-03-01T00%3A34%3A20Z&se=2019-04-01T00%3A39%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[68a4b22](https://github.com/Azure/portaldocs/pull/191/commits/68a4b22de3364916c50d0472e6594918f82894b4)_: 

:clock10: **Preparing**: average preparing time is _45 sec(s)_

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[68a4b22](https://github.com/Azure/portaldocs/pull/191/commits/68a4b22de3364916c50d0472e6594918f82894b4)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C3d86255f-f488-188e-02a3-9e92aa67830d%5CPullRequest%5C201903010040322985-191%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rGN9S1XHNDhNZUWl6gzjUnhHR%2Bjq1L1Ktg9RHVDqB5c%3D&st=2019-03-01T00%3A35%3A43Z&se=2019-04-01T00%3A40%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**